schemas/common_enums.proto

```diff
@@ -43,3 +43,8 @@ enum VariableType {
   BYTES = 6;
   NULL = 7;
 }
+
+enum WaitForThreadsFailureStrategy{
```

Comment:
@coltmcnealy-lh i want your opinion on this naming thing. basically, `ALL_NODES` means wait to all nodes to be completed. and `SINGLE_NODE` means once a single node fails, it will stop the entire wfrun

Reply:
I like `WaitForThreadsStrategy` you are right this makes more sense in this context and could support future features like the one you are mentioning. but. specifically for the strategy `FAIL_FAST` i would like to suggest `FAIL_AFTER_ONE`.